django/db/models/fields/__init__.py

```diff
@@ -239,6 +241,13 @@ def _check_field_name(self):
         else:
             return []
 
+    @property
+    def rel(self):
+        warnings.warn(
+            "Usage of field.rel has been deprecated. Use field.remote_field instead.",
+            RemovedInDjango20Warning, 2)
```

Comment:
We could do an accelerated deprecation as this wasn't ever public API. But I don't see a reason for that. Should I document the internal API deprecations?